### PR TITLE
Tom & Jerry destroyed media, lesss go!

### PR DESCRIPTION
**Pull request by thomas065**

### CODE REVIEW

File: backend/controllers/mediaController.js [hunk 1]
```diff
@@ -6,31 +6,31 @@ const getMedia = async (req, res, next) => {
...
+        .json({message: 'Created media', media});
+    } catch (err) {
         next(err)
     }
 }
 
 const deleteMedia = async (req, res, next) => {
     try {
-        const media = await Media.deleteMedia();
+        const media = await Media.deleteOne();
```
Comment (hjkmines) [inline backend/controllers/mediaController.js:29]: should be `deleteMany()`

**Review by kristenking — APPROVED** [review]
We met with Jerry and went through the code review together. Everything looks good and clean. Approved. Thank you for the hard work, Thomas and Jerry!

File: backend/controllers/mediaController.js (hunk 1 above)
Comment (thomas065) [reply backend/controllers/mediaController.js]: All changed and ready for approval.

**Comment by thomas065** [comment]
updated code to include route for /:mediaId

**Review by hjkmines — APPROVED** [review]

**Pull request merged**